src/pages/course/CourseList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import {
+    Datagrid,
+    EditButton,
+    Filter,
+    List,
+    NumberInput,
+    Responsive,
+    SimpleList,
+    TextField,
+    TextInput,
+} from 'react-admin';
+
+const CourseFilter = (props) => (
+    <Filter {...props}>
+        {/* <TextInput label="Search" source="q" alwaysOn /> */}
```

Comment:
Not sure if you've commented these for future reference but if not it would be good to remove these